rakelib/utils.rake

```diff
@@ -68,6 +68,7 @@ class Utils
 
   # select the xcode version we want/support
   def self.version_select
+    Rake.sh 'mdimport /Applications/Xcode* && sleep 1'
```

Comment:
🤔, I see.

I agree that `sleep 1` is a hack that won't fix the problem in a definitive way, and it looks like this spotlight problem is a bug on [10.12.4](https://openradar.appspot.com/31646293).

Enable spotlight was the correct fix for CircleCI (as @DanToml mentioned), so we still need a workaround to Sierra bug. It's plausible to expect that the next Sierra update will fix this bug, so I think that might be unnecessary to have a fallback method to spotlight that will be necessary just for a couple of months.

So I have a proposal that would be more assertive, what if remove the sleep and update the error message to something like: 

> SwiftGen requires Xcode 8.x, but we were not able not found it. If it's already installed update your Spotlight index with "mdimport /Applications/Xcode*"

It will work reliable for most of the cases without any hacks and we inform the user a way to fix if an error happens.

What do you think?